catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/CswRecordConverter.java

```diff
@@ -223,6 +228,31 @@ public Metacard transform(InputStream inputStream, String id)
     @Override
     public BinaryContent transform(Metacard metacard, Map<String, Serializable> arguments)
             throws CatalogTransformerException {
+        if (StringUtils.isNotBlank(metacard.getMetadata())) {
+            // Check if the metadata is csw:Record
+            try {
+                StringReader xml = new StringReader(metacard.getMetadata());
+                XMLEventReader reader = factory.createXMLEventReader(xml);
+                boolean rootFound = false;
+                while (reader.hasNext() && !rootFound) {
+                    XMLEvent event = reader.nextEvent();
+                    if (event.isStartElement()) {
+                        rootFound = true;
+                        QName name = event.asStartElement()
+                                .getName();
+                        if (StringUtils.equals(CswConstants.CSW_RECORD_LOCAL_NAME,
+                                name.getLocalPart())
+                                && StringUtils.equals(CswConstants.CSW_OUTPUT_SCHEMA,
+                                name.getNamespaceURI())) {
+                            return new BinaryContentImpl(IOUtils.toInputStream(metacard.getMetadata()),
+                                    new MimeType());
+                        }
+                    }
+                }
+            } catch (Exception e) {
+                // Ignore and proceed with the transform.
```

Comment:
I guess if you really want one.... but then on a large result set you could see that log statement a bunch and then it may lose it's value.